packages/google_maps_flutter/lib/src/controller.dart

```diff
@@ -236,4 +236,25 @@ class GoogleMapController {
         'map#getLatLng', screenCoordinate._toJson());
     return LatLng(latLng[0], latLng[1]);
   }
+
+  /// Show [Marker] info window
```

Comment:
This seems to qualify as a "useless doc" in Flutter's style guide. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation
How about adding more information to this method that people reading it might not know about?
An example:
```
/// Programmatically show the Info Window for a marker.
///
/// The `markerId` must match one of the markers on the map.
/// An invalid `markerId` triggers an "Invalid markerId" error.
///
/// * See also:
///   * [hideMarkerInfoWindow] to hide the Info Window.
///   * [isMarkerInfoWindowShown] to check if the Info Window is showing.
```